README.md

```diff
@@ -23,4 +23,39 @@ The crawler can be started using the start script:
 
 ```shell script
 ./scripts/start.sh
-```
\ No newline at end of file
+```
+## Note
+
+Ensure your server has at least 3GB of memory as the Elasticsearch stack docker will require 2GB.
+
+## Prepare the feeder process
+
```

Comment:
Add a little explanation:

The feeder process is used to send the first URL to the crawler.